modules/core-updates/8.0.md

```diff
@@ -15,16 +15,25 @@ menuTitle: Changes in 8.0
 
 ## PHP support
 
-PrestaShop 8.0 adds supports for PHP 8.0 and requires at least PHP 7.2.5.
+PrestaShop 8.0 adds supports for PHP 8.0 and PHP 8.1 and requires at least PHP 7.2.5.
 
 ## Notable changes
 
 * Javascript files for migrated pages in the Back Office [now are written in TypeScript](https://github.com/PrestaShop/PrestaShop/pull/23221) (related change: [#24791](https://github.com/PrestaShop/PrestaShop/pull/24791)).
 * Support for [legacy translation dictionaries][legacy-translation] (eg: `en.php`, `fr.php`..., deprecated in 1.7), [has been removed](https://github.com/PrestaShop/PrestaShop/pull/24401) from Core controllers. Only modules can still use this system now.
-* All modules present in the `modules` directory during the installation process will be installed by default. Modules can use `Module::postInstall()` to execute code after the shop installation is done ([Read more](https://github.com/PrestaShop/PrestaShop/pull/24676)).
+* All modules present in the `modules` directory during the installation process will be installed by default [unless you choose otherwise](https://github.com/PrestaShop/PrestaShop/pull/27099), if you install from the browser. Modules can use `Module::postInstall()` to execute code after the shop installation is done ([Read more](https://github.com/PrestaShop/PrestaShop/pull/24676)).
 * Twig has been updated to version 3. Deprecated features from [v1](https://twig.symfony.com/doc/1.x/deprecated.html) & [v2](https://twig.symfony.com/doc/2.x/deprecated.html) are no longer available.
 * All upgrade scripts have been [removed from the Core](https://github.com/PrestaShop/PrestaShop/pull/25794) and moved to the [Upgrade module](https://github.com/PrestaShop/autoupgrade).
 * Using a hook alias will trigger a deprecation notice.
+* Historically the default Carrier's name was `0`, [now](https://github.com/PrestaShop/PrestaShop/pull/27439) it is `Click and collect`
+* Live exchange rate feature [has been removed](https://github.com/PrestaShop/PrestaShop/pull/28177) from the core
+* In back office, the .rtlfix file [has been removed](https://github.com/PrestaShop/PrestaShop/pull/28277). It means that if you use it, you'll need to port your changes inside the SCSS stylesheet.
+* Version 8 comes with a [new Distribution API](https://github.com/PrestaShop/PrestaShop/pull/27632), and native modules are now available from the API dedicated to open source projects with a possibility to include your source of modules in the PrestaShop
+* Because of the [support for PHP 8.1](https://github.com/PrestaShop/PrestaShop/pull/28402) a lot of methods have return types now
```

Comment:
```suggestion
* Because of the [support for PHP 8.1](https://github.com/PrestaShop/PrestaShop/pull/28402) many methods have declare a return types now.
```